docker/Jenkinsfile-build-k8s-test-docker

```diff
@@ -0,0 +1,138 @@
+#! /usr/bin/groovy
+
+final String NODE_LABEL = 'docker && !mr-0xc8 && !micro'
+final String DOCKER_STASH = 'h2o-3-k8s-docker-stash'
+final String REGISTRY_PREFIX = "${params.dockerRegistry}"
+final String IMAGE_NAME_PREFIX = "opsh2oai/h2o-3-k8s"
+final String IMAGE_NAME_PREFIX_RUNNER = "opsh2oai/h2o-3-k8s-runner"
+
+def pipelineContext = null
+def version = null
+
+properties(
+    [
+        parameters(
+            [
+                string(defaultValue: 'master', description: 'Branch to checkout', name: 'gitBranch'),
+                string(name: 'dockerRegistry', defaultValue: 'harbor.h2o.ai'),
+                booleanParam(name: 'force', defaultValue: false, description: 'If false and image with version specified by BuildConfig exists in repository, then the build fails.')
+            ]
+        )
+    ]
+)
+
+node(NODE_LABEL) {
+    final String stageName = 'Checkout and Prepare'
+    stage(stageName) {
+        def scmEnv = git credentialsId: 'c6bab81a-6bb5-4497-9ec9-285ef5db36ea',
+                poll: false,
+                url: 'https://github.com/h2oai/h2o-3',
+                branch: params.gitBranch
+
+        def pipelineContextFactory = load('scripts/jenkins/groovy/pipelineContext.groovy')
+        pipelineContext = pipelineContextFactory('.', 'MODE_HADOOP', scmEnv, true)
```

Comment:
What are the implications of using "MODE_HADOOP"?

Reply:
This should be removed, copy & paste leftover.

In `buildConfig.groovy`, this triggers hadoop build: `    this.buildHadoop = (mode == 'MODE_HADOOP' || mode == 'MODE_KERBEROS')`

Reply:
Removed in https://github.com/h2oai/h2o-3/pull/4439/commits/6528a530c526de7484d074f55bd3d8a3ec33e611

Reply:
Why are we building hadoop jars when running on k8s?

Reply:
We're not anymore. Copy and paste error.